src/Modules/Footer/FooterComponent.js

```diff
@@ -38,11 +38,11 @@ const FooterComponent = () => (
             </ul>
           </div>
           <div className="col-md-4">
-            <span className="list-title">FOR FOODBANKS</span>
+            <span className="list-title">For Foodbanks & Agencies</span>
             <ul>
               <li>
                 <Link to={RENDER_URL.FRESHTRAK_WORKING}>
-                  Working with FreshTrak
+                  FreshTrak: Partners
```

Comment:
OnClick `FreshTrak: Partners` link, should it redirect to `https://freshtrak.com/freshtrak-working` or `https://partners.freshtrak.com/`

Reply:
Added